vhdbuilder/packer/install-dependencies.sh

```diff
@@ -638,6 +638,7 @@ mcr.microsoft.com/azure-application-gateway/kubernetes-ingress:1.2.0
 mcr.microsoft.com/azure-application-gateway/kubernetes-ingress:1.3.0
 mcr.microsoft.com/oss/azure/aad-pod-identity/nmi:v1.6.3
```

Comment:
I have a question, @xuto2  @bowang-666 should we remove the old version images here?

Reply:
if there is no overlay pods that will need to access this image then we should. In fact we want that as the disk  space is very limited(30GB) and we often need to clean up old images, but it's hard for us to do it for the addons, so yes please remove the old, unnecessary images as much as you can

Reply:
thanks for the confirmation. I had removed the 1.6.3 one. Will remove the 1.7.0 one after the new version has been fully rolled out.